RELEASE.md

```diff
@@ -17,7 +23,7 @@ Please follow the established format:
 - Fix to search for a '<lambda' Python function in the sidebar. (#1497)
 - Add favicon to Kedro-Viz. (#1509)
 - Remove python upper-bound requirements and add KedroVizPythonVersion warning. (#1506)
-
+- Resolved various bugs affecting server-side rendering, updated dependencies to ensure compatibility with Vite and Next.js environments (#1508)
```

Comment:
Why is this here? That [PR](https://github.com/kedro-org/kedro-viz/pull/1508) was never merged, right?

Reply:
+1